fuse_models/test/test_unicycle_2d_predict.cpp

```diff
@@ -344,7 +345,19 @@ TEST(Predict, predictJacobians)
   double acc_linear2_x = 0.0;
   double acc_linear2_y = 0.0;
 
-  fuse_core::Matrix8d J;
+  const std::array<size_t, 5> block_sizes = {2, 1, 2, 1, 2};
```

Comment:
Updating so the full jacobian matrix is properly converted into the layout `ceres` expects. This is a conversion similar to the one mentioned before in on of the `predict` free function. In this case the `num_parameter_blocks` is a compile time constant, so we can use an `std::array` to store the jacobians per parameter block. In the `predict` method is the same, although there we use some `static` variables.

If we encapsulate the conversion somewhere else, I think it'd have to be a templated class, so we can indicate the parameter block sizes. Thoughts?